scripts/check_versions.sh

```diff
@@ -3,7 +3,7 @@
 # Use, adjust, copy/paste, etc. as necessary to answer your questions.
 # This may be helpful when updating dependency versions in CI.
 # Tip: add `--json` for more information.
-conda search 'numpy[channel=conda-forge]>=1.24.1'
+conda search 'numpy[channel=conda-forge]>=1.24.2'
```

Comment:
Interesting, didn't know about this syntax for channel. I used `-c` syntax before: `conda search -c conda-forge "numpy>=1"`.

Reply:
Yeah, using `-c conda-forge` _adds_ the conda-forge channel to search in, but will still search in default channels. The syntax above only searches in the specified channel.

Scroll to the bottom here: https://docs.conda.io/projects/conda/en/latest/commands/search.html